pkg/operator/ceph/nfs/controller.go

```diff
@@ -279,12 +279,6 @@ func (r *ReconcileCephNFS) reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, errors.Wrapf(err, "invalid ceph nfs %q arguments", cephNFS.Name)
 	}
 
-	// Always create the default pool
-	err = r.createDefaultNFSRADOSPool(cephNFS)
-	if err != nil {
-		return reconcile.Result{}, errors.Wrapf(err, "failed to create default pool %q", cephNFS.Spec.RADOS.Pool)
-	}
-
```

Comment:
We do I think, at minimum, want to verify that the pool exists. Though I still think that creating it might be good. That is, unless the lower level `ceph osd pool create .nfs` resets a customized pool back to default values. In that case, maybe it'll be easier to chat in huddle.

Reply:
I'll add a check for the existence of the pool